lms/views/api/blackboard/_schemas.py

```diff
@@ -17,6 +17,7 @@ class Meta:
     id = fields.Str(required=True)
     display_name = fields.Str(data_key="name", required=True)
     updated_at = fields.Str(data_key="modified", required=True)
+    mime_type = fields.Str(data_key="mimeType")
```

Comment:
I've made `mime_type` an optional field. If it's missing we'll just assume the file is not a PDF. Folders in Blackboard API responses don't have `mime_type`'s so making the field optional will be useful / necessary in future when we add folder support.